CONTRIBUTING.md

```diff
@@ -1 +1 @@
-See the [contributing guide](https://github.com/pulsar-edit/.github/blob/main/CONTRIBUTING.md)
+See the [Pulsar contributing guide](https://github.com/pulsar-edit/.github/blob/main/CONTRIBUTING.md)
```

Comment:
```suggestion
See how you can contribute [here](https://github.com/pulsar-edit/.github/blob/main/CONTRIBUTING.md)
```
I originally removed the word Atom/Pulsar as it's redundant. However I think we should just adjust all of these to the above verbiage. What do we think @pulsar-edit/documentation?

Reply:
The changed verbiage is actually very nice, and allows anyone after us to avoid having endless rebranding to do. Personally I am totally fine with it, and would be happy to see it rolled out wherever possible, as its simpler and more inviting.

Reply:
I like this new wording!